client/src/components/Dropdown/Dropdown.js

```diff
@@ -1,24 +1,51 @@
 import React, { Component } from 'react';
-import classnames from 'classnames';
+import classNames from 'classnames';
 
 import { DropdownIcon } from './DropdownIcon';
 import { DropdownPlus } from './DropdownPlus';
 
 class Dropdown extends Component {
-  state = {
-    active: false,
-  };
+  constructor(props) {
+    super(props);
+    this.node = React.createRef();
+    this.state = { active: false };
+  }
 
-  handleClick = e => {
-    this.setState(prevProps => ({
-      active: !prevProps.active,
-    }));
+  handleClick = event => {
+    console.log('handle click', this.node.contains(event.target));
```

Comment:
Let's remove the `console.log` functions in throughout this function. You can always add them back in locally if you need to debug, but removing them from here will help keep the console log clean so it doesn't get filled with too much info.